src/emulator/gui/src/gui.cpp

```diff
@@ -120,12 +123,40 @@ static void init_font(State &gui) {
     gui.normal_font = io.Fonts->AddFontFromMemoryTTF(gui.font_data.data(), font_file_size, 16, &font_config, io.Fonts->GetGlyphRangesJapanese());
 }
 
+static void init_background(State &gui, const std::string &image_path) {
+    if (!std::ifstream(image_path).good()) {
+        LOG_INFO("Invalid background file path {}.", image_path);
+        return;
+    }
+
+    int32_t width = 0;
+    int32_t height = 0;
+    stbi_uc *data = stbi_load(image_path.c_str(), &width, &height, nullptr, 3);
```

Comment:
STBI_rgb_alpha. Waste some more GPU memory to acheive godly effect on some image